.gitlab-ci.yml

```diff
@@ -937,6 +937,18 @@ twistlock_scan:
     - notary -d ~/.docker/trust key import /tmp/docker.key; rm /tmp/docker.key
   dependencies: [] # Don't download Gitlab artefacts
 
+.docker_delete_tag_job_definition: &docker_delete_tag_job_definition
+  stage: image_deploy
+  tags: [ "runner:docker", "size:large" ]
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/docker-notary:0.6.1
```

Comment:
You don't need Notary here, amirite?

Reply:
Good catch, we're not signing anything so yes we don't need Notary here, do we have an alternative docker image we can use instead ? (with `aws` command installed and configured, `python` and `curl` installed)

Reply:
IDK, but there should be :)